examples/c++/ndarray_example.cpp

```diff
@@ -4,12 +4,13 @@
 #include <nda/nda.hpp>
 #include <nda/h5.hpp>
 
-using namespace std::literals;
+using namespace std::complex_literals;
```

Comment:
Doesn't this break the use of string literal below? "/input.h5"s

Reply:
Fair point I reverted it, but somehow it worked.